### ISSUE TITLE
Problem with output

### ISSUE THREAD

**Issue by risingphoenix619** [otenim/GLCIC-PyTorch#4]
Hi, i ran your demo using the pre-trained model, but the result were not very accurate.

![out](https://user-images.githubusercontent.com/47709394/52909789-06237580-32b4-11e9-9cd9-fdec411d7b19.jpg)

Following are the version of packages i am using:
python 3.6.0
scipy 1.1.0
numpy 1.14.5
pyamg 3.3.2
pillow 4.2.1
pytorch 0.4.1
torchvision 0.2.1
tqdm 4.24.0

All the packages were downloaded using conda and demo was ran in conda environment.

**Comment by otenim** [comment]
I think there are potentially two problems in my repo.

* As written in my repo, the pretrained model we share now is trained with much smaller batch size (==16) than the paper setting (batch size == 96) due to lack of GPU memory.
Currently we are training a model with exactly the same batch size using 4 GPUs.
Please wait little bit more.

* I think there might be some problems in poissonblending.py.
Some code lines in this script are deprecated in latest versions.
If the batch size change didn't work for the low accuracy problem,
I would check poissonblending.py as the next task.
